.env

```
@@ -1,6 +1,6 @@
 GANACHE_MNEMONIC="gadget evoke oil anchor able cake lift drop fork useful once owner"
-ACCOUNT_PRIVATE_KEY=ACCOUNT_PRIVATE_KEY=0x2dd714e4360ea753e9dfc18bbd9a59414d9a5308448c664d8e25c1e07b9b0292  
-
+BLOCKCHAIN_URL=http://127.0.0.1:7545
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BLOCKCHAIN_URL key should go before the GANACHE_MNEMONIC key

(UnorderedKey)